<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.pfl.automation.org62.testcases.TS03_lightning" tests="5" time="0.001" ignored="0" failures="0" timestamp="18 May 2019 13:27:49 GMT" skipped="4" hostname="Deepak-Sharma1" errors="1">
  <testcase name="TS03_DataReset" time="0.000" classname="com.pfl.automation.org62.testcases.TS03_lightning">
    <skipped/>
  </testcase> <!-- TS03_DataReset -->
  <testcase name="TS03_01_verifying_contact_address_send_swag" time="0.000" classname="com.pfl.automation.org62.testcases.TS03_lightning">
    <skipped/>
  </testcase> <!-- TS03_01_verifying_contact_address_send_swag -->
  <testcase name="TS03_02_verifying_account_address_end_swag" time="0.001" classname="com.pfl.automation.org62.testcases.TS03_lightning">
    <skipped/>
  </testcase> <!-- TS03_02_verifying_account_address_end_swag -->
  <testcase name="LogIn" time="0.000" classname="com.pfl.automation.org62.testcases.TS03_lightning">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.pfl.automation.base.BaseTest.LogIn(BaseTest.java:130)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:590)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:295)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:84)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:90)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:203)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:155)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:103)
]]>
    </error>
  </testcase> <!-- LogIn -->
  <testcase name="TS03_DataSet_Creation" time="0.000" classname="com.pfl.automation.org62.testcases.TS03_lightning">
    <skipped/>
  </testcase> <!-- TS03_DataSet_Creation -->
</testsuite> <!-- com.pfl.automation.org62.testcases.TS03_lightning -->
